### PR TITLE
https://issues.redhat.com/browse/ACM-11010 2.10

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-11010

### CODE REVIEW

File: governance/compliance_operator_policy.adoc [hunk 1]
```diff
@@ -1,12 +1,19 @@
 [#compliance-operator-policy]
-= Compliance operator policy
+= Compliance Operator policy
 
-Compliance operator is an operator that runs OpenSCAP and allows you to keep your {ocp} cluster compliant with the security benchmark that you need. You can install the compliance operator on your managed cluster by using the compliance operator policy. 
+The Compliance Operator assists users by automating the inspection of numerous technical implementations and compares those against certain aspects of industry standards, benchmarks, and baselines; the Compliance Operator is not  an auditor. In order to be compliant or certified within these various standards, you need to engage an authorized auditor such as a Qualified Security Assessor (QSA), Joint Authorization Board (JAB), or other industry recognized regulatory authority to assess your environment.
```
Comment (dockerymick) [inline governance/compliance_operator_policy.adoc]: Suggestion after revised by Brandi

```suggestion
You can use the Compliance Operator to automate the inspection of numerous technical implementations and compare those against certain aspects of industry standards, benchmarks, and baselines. The Compliance Operator is not an auditor. To be compliant or certified with these various standards, you need to engage an authorized auditor such as a Qualified Security Assessor (QSA), Joint Authorization Board (JAB), or other industry recognized regulatory authority to assess your environment.  

Recommendations that are generated from the Compliance Operator are based on generally available information and practices regarding such standards, and might assist you with remediations, but actual compliance is your responsibility. Work with an authorized auditor to achieve compliance with a standard.

For the latest updates, see the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/security_and_compliance/compliance-operator#compliance-operator-release-notes[Compliance Operator release notes].
```

File: governance/compliance_operator_policy.adoc [hunk 2]
```diff
@@ -1,12 +1,19 @@
...
 
-The compliance operator policy is created as a Kubernetes configuration policy in {product-title-short}. {ocp-short} supports the compliance operator policy. For more information, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html-single/security_and_compliance/index#compliance-operator-understanding[Managing the Compliance Operator] in the {ocp-short} documentation for more details.
+The Compliance Operator makes recommendations based on generally available information and practices regarding such standards and may assist with remediations, but actual compliance is your responsibility. You are required to work with an authorized auditor to achieve compliance with a standard.
 
+For the latest updates, see the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/security_and_compliance/compliance-operator#compliance-operator-release-notes[Compliance Operator release notes].
```
Comment (dockerymick) [inline governance/compliance_operator_policy.adoc]:
```suggestion
```

**Review by dockerymick — COMMENTED** [review]
added suggestions based on the comments from the issue

**Review by swopebe — APPROVED** [review]

File: governance/compliance_operator_policy.adoc [hunk 3]
```diff
@@ -1,12 +1,23 @@
...
+Recommendations that are generated from the Compliance Operator are based on generally available information and practices regarding such standards, and might assist you with remediations, but actual compliance is your responsibility. Work with an authorized auditor to achieve compliance with a standard.
 
+For the latest updates, see the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/security_and_compliance/compliance-operator#compliance-operator-release-notes[Compliance Operator release notes].
+
+The Compliance Operator makes recommendations based on generally available information and practices regarding such standards and may assist with remediations, but actual compliance is your responsibility. You are required to work with an authorized auditor to achieve compliance with a standard.
```
Comment (dockerymick) [inline governance/compliance_operator_policy.adoc]:
```suggestion
```

File: governance/compliance_operator_policy.adoc [hunk 4]
```diff
@@ -1,12 +1,20 @@
...
 
+
+For the latest updates, see the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/security_and_compliance/compliance-operator#compliance-operator-release-notes[Compliance Operator release notes].
+
+
+[#compliance-operator-policy-overview]
+== Compliance Operator policy overview
+
+You can install the Compliance Operator on your managed cluster by using the Compliance Operator policy. The Compliance operator policy is created as a Kubernetes configuration policy in {product-title-short}. {ocp-short} supports the compliance operator policy.
+//is this note valid? who can confirm this? Is it important to mention which architecture do not support the Compliance Operator?
```
Comment (dockerymick) [inline governance/compliance_operator_policy.adoc:17]:
```suggestion

//is this note valid? who can confirm this? Is it important to mention which architecture do not support the Compliance Operator?
```

**Pull request merged**